src/vec_int128_ppc.h

```diff
@@ -121,56 +142,25 @@
  * See \ref mainpage_sub_1_3 for more background on extended quadword
  * computation.
  *
+ * \section int128_perf_0_0 Performance data.
+ * High level performance estimates are provided as an aid to function
+ * selection when evaluating algorithms. For background on how
+ * <I>Latency</I> and <I>Throughput</I> are derived see:
+ * \ref perf_data
  */
 
-/** \brief Vector Add Unsigned Quadword Modulo.
- *
- *	Add two vector __int128 values and return result modulo 128-bits.
- *
- *	@param a 128-bit vector treated a __int128.
- *	@param b 128-bit vector treated a __int128.
- *	@return __int128 sum of a and b.
- */
-static inline vui128_t
-vec_adduqm (vui128_t a, vui128_t b)
-{
-  vui32_t t;
-#ifdef _ARCH_PWR8
-#ifndef vec_vadduqm
-  __asm__(
-      "vadduqm %0,%1,%2;"
-      : "=v" (t)
-      : "v" (a),
-      "v" (b)
-      : );
-#else
-  t = (vui32_t) vec_vadduqm (a, b);
-#endif
-#else
-  vui32_t c, c2;
-  vui32_t z= { 0,0,0,0};
-
-  c = vec_vaddcuw ((vui32_t)a, (vui32_t)b);
-  t = vec_vadduwm ((vui32_t)a, (vui32_t)b);
-  c = vec_sld (c, z, 4);
-  c2 = vec_vaddcuw (t, c);
-  t = vec_vadduwm (t, c);
-  c = vec_sld (c2, z, 4);
-  c2 = vec_vaddcuw (t, c);
-  t = vec_vadduwm (t, c);
-  c = vec_sld (c2, z, 4);
-  t = vec_vadduwm (t, c);
-#endif
-  return ((vui128_t) t);
-}
-
 /** \brief Vector Add & write Carry Unsigned Quadword.
  *
- *	Add two vector __int128 values and return the carry out.
+ *  Add two vector __int128 values and return the carry out.
+ *
+ *  |processor|Latency|Throughput|
+ *  |--------:|:-----:|:---------|
+ *  |power8   | 4     |2/2 cycles|
+ *  |power9   | 3     | 2/cycle  |
  *
- *	@param a 128-bit vector treated a __int128.
- *	@param b 128-bit vector treated a __int128.
- *	@return __int128 carry of the sum of a and b.
+ *  @param a 128-bit vector treated a __int128.
```

Comment:
This should say `treated as` here.
Global comment.  It seems to have snuck in somewhere, then copied and pasted.

Reply:
This is a necessary doxygen fix. Seems doxygen gets confused if the indentation of the brief description and @param/@return are not aligned. This is correcting the indentation so that doxygen will format correctly.

As for the size of the change, it is hard to break these down as the compares are all related as variations on the theme, 

Also in my defense I provided a changelog that explicitly lists the New functions. Could have just reviewed those.

Reply:
I'll commit these and submit the unit and perf tests by monday.